src/auth/ha-auth-flow.ts

```diff
@@ -212,6 +242,10 @@ class HaAuthFlow extends litLocalizeLiteMixin(LitElement) {
       this._stepData = stepData;
     }
 
+    if (step.type === "external") {
+      this._externalStep(step);
+    }
+
```

Comment:
This could perhaps be moved into the postFlow function same as redirect check, but i'm unsure we need should wait for the updateComplete stuff or not here.